crates/core/src/db/relational_db.rs

```diff
@@ -477,7 +467,7 @@ impl RelationalDB {
     ///
     /// NOTE: It loads the data from the table into it before returning
     #[tracing::instrument(skip(self, tx, index), fields(index=index.name))]
-    pub fn create_index(&self, tx: &mut MutTxId, index: IndexDef) -> Result<IndexId, DBError> {
+    pub fn create_index(&self, tx: &mut MutTxId, table_id: TableId, index: IndexDef) -> Result<IndexId, DBError> {
```

Comment:
Why is `table_id` plumbed through but not used? Does it get used in #267 ?

Reply:
Correct. I will remove this change because if is not used is pointless here.